### PR TITLE
Use english messages as message ids

### PR DESCRIPTION
**Pull request by dpoirier**
## Description
Use English messages as message IDs for better interoperability with Transifex, Django, and the rest of the translating world.

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -144,55 +144,70 @@ Alternatively, you can deploy the project using Dokku. See the
 How the license translation is implemented
 ------------------------------------------
 
-First, note that translation is done in two different ways. Most things use the built-in
+First, note that translation is done use two sets of files. Most things use the built-in
```
Comment (Audiosutras) [inline README.rst:147]:
```suggestion
First, note that translation uses two sets of files. Most things use the built-in
```

File: licenses/management/commands/load_html_files.py [hunk 2]
```diff
@@ -12,6 +12,14 @@
 from licenses.utils import parse_legalcode_filename, validate_dictionary_is_all_text
 
 
+def clean_string(s):
```
Comment (Audiosutras) [inline licenses/management/commands/load_html_files.py:15]: Should we move `clean_string` into `licenses.utils`?

**Review by Audiosutras — COMMENTED** [review]
I dropped `cc_licenses` db, re-created it, and ran migrations.

When I run:
```
python manage.py load_html_files ../creativecommons.org/docroot/legalcode
```
I get the error:
```
KeyError: 's1_definitions_you2'
```
The error occurs on `line 181` of `load_html_files.py`

**Comment by Audiosutras** [comment]
For the `breadcrumb` and `menu-sidebar`, we need to update the title to use `{% include "includes/snippet/license_medium_title.html" %}`

**Comment by Audiosutras** [comment]
As a part of this PR, can we also start to populate `translation_last_update` on the `LegalCode` model?

**Comment by dpoirier** [comment]
> I dropped `cc_licenses` db, re-created it, and ran migrations.
> 
> When I run:
> 
> ```
> python manage.py load_html_files ../creativecommons.org/docroot/legalcode
> ```
> 
> I get the error:
> 
> ```
> KeyError: 's1_definitions_you2'
> ```
> 
> The error occurs on `line 181` of `load_html_files.py`.

This is working for me, but there have been recent updates to creativecommons.org. Maybe try pulling the latest version of that and try again?

**Comment by dpoirier** [comment]
> As a part of this PR, can we also start to populate `translation_last_update` on the `LegalCode` model?

If we want to get the last translation change date from Transifex, we'll need to use the Transifex API. I think we'll have to get that date anyway in part 2 of CC-31, when we're looking for updated translations in Transifex.

**Comment by dpoirier** [comment]
> For the `breadcrumb` and `menu-sidebar`, we need to update the title to use `{% include "includes/snippet/license_medium_title.html" %}`

Good catches, will do.

**Review by Audiosutras — APPROVED** [review]
💯

**Pull request merged**